tests/model/test_scvi.py

```diff
@@ -1297,3 +1297,57 @@ def test_scvi_num_workers():
     model.get_reconstruction_error()
     model.get_normalized_expression(transform_batch="batch_1")
     model.get_normalized_expression(n_samples=2)
+
```

Comment:
Same here. Just have one test for ddp and not in the test_scvi thing.

Reply:
ok

Reply:
we can add a label multi-gpu to test those kind of tests